### ISSUE TITLE
When importing OpenAPI spec use tags versus paths

### ISSUE THREAD

**Issue by joshuawoodward** [postmanlabs/postman-app-support#3892]
If you import a OpenAPI (swagger) spec file, the collection and folders appear to be grouped by the paths, versus the tags.

In a Swagger UI endpoints are grouped and ordered by the Tags
![2017-12-01_13-49-13](https://user-images.githubusercontent.com/1133928/33504995-a73a5ede-d69e-11e7-9ac5-c741d2f19ac2.png)

In Postman the endpoints are grouped and ordered by the paths (just the root)
![2017-12-01_13-49-31](https://user-images.githubusercontent.com/1133928/33505026-bf0d78fc-d69e-11e7-95e4-7b96b166fa23.png)

Ideally postman would look like the following (after I manually edited). This would match the grouping of our documentation (not postman docs) and our swagger UI (playground)

![2017-12-01_13-54-34](https://user-images.githubusercontent.com/1133928/33505198-4224670a-d69f-11e7-9563-76e9c6aef247.png)

**Comment by mshimao** [comment]
+1

**Comment by abhijitkane** [comment]
@joshuawoodward @mshimao This was actually a decision we made while working on the Swagger->Postman conversion. A few areas where Postman differs w.r.t tags:

1. Swagger lets you have multiple tags per operation, allowing a request to show up in multiple folders. This is currently forbidden in Postman. We'll need to take a deeper look at our folder/request structure, but I don't see this changing anytime soon.
2. I'm not sure how Swagger would let you define have multiple levels of folders. This is something that Postman supports (even though the Swagger2.0 importer doesn't yet - we'll be updating this soon), and working with just one level of folders is prohibitive for large APIs.

We're working on a tagging system to better mirror what Swagger offers - we'll keep this thread updated with more info.

**Comment by jgroom33** [comment]
An acceptable implementation for the use cases I've encountered would be to use the first tag noted in the request.  This would allow people to reorder the tag list for each request to achieve the desired outcome.

**Comment by benhamill** [comment]
I agree with @jgroom33. We use the tags for organization of endpoints logically in a way that isn't necessarily related to the URLs. Using the first tag would be what I'd expect and is, I'm pretty sure, what other tools that use OpenAPI do (e.g. redoc).

**Comment by smshuja** [comment]
ReDoc is using vendor extensions
https://github.com/Rebilly/ReDoc/blob/master/docs/redoc-vendor-extensions.md#x-tagGroups

This seems like a good option.

**Comment by a85** [comment]
We'll be moving all these issues to importer specific projects. These are open-source plug-ins that Postman maintains so feel free to submit a pull-request and we can look at the behavior. We will be migrating these issues to that tracker soon.
